{"name": "test_login_with_valid_email_and_invalid_password_Duplicate", "status": "passed", "start": 1695710686204, "stop": 1695710689153, "uuid": "ef0de10c-0f6d-4f94-a623-2a6b8d4f6f57", "historyId": "9581a2f8ea7f6829aae721e266b65c65", "testCaseId": "9581a2f8ea7f6829aae721e266b65c65", "fullName": "Tests.test_Login(duplicate).TestPanarayLoginDuplicate#test_login_with_valid_email_and_invalid_password_Duplicate", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login(duplicate)"}, {"name": "subSuite", "value": "TestPanarayLoginDuplicate"}, {"name": "host", "value": "WONIT11L32CV6Z"}, {"name": "thread", "value": "4252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login(duplicate)"}]}